Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml

```diff
@@ -55,22 +55,6 @@
                             <li><h:link outcome="tasks" value="#{msgs.allMyTasks} (#{tasks.size()})"/></li>
                         </ul>
                     </li>
-                    <li>
-                        <!-- List: active users -->
-                        <a href="#"><i class="fa fa-user-o fa-lg"/><h:panelGroup
-                            rendered="#{SessionForm.activeSessions.size() gt 0}"
-                            class="batch-count bc-info">#{SessionForm.activeSessions.size()}</h:panelGroup></a>
-                        <ul>
-                            <li><h3>#{msgs.activeUsers}</h3></li>
-                            <ui:repeat
-                                value="#{SessionForm.activeSessions.subList(0, (SessionForm.activeSessions.size() gt 3) ? 3 : SessionForm.activeSessions.size())}"
-                                var="activeSession">
-                                <li><div><span class="nav-item-col">#{activeSession['userName']}</span></div></li>
-                            </ui:repeat>
-                            <li><h:link outcome="activeUsers"
-                                        value="#{msgs.allActiveUsers} (#{SessionForm.activeSessions.size()})"/></li>
```

Comment:
Probably this message is right now not used anymore an can be removed.

Reply:
Removed.